stable/traefik/templates/deployment.yaml

```diff
@@ -83,6 +83,7 @@ spec:
           periodSeconds: 10
           successThreshold: 1
           timeoutSeconds: 2
+        hostNetwork: {{ .Values.deployment.hostNetwork }}
```

Comment:
This entire setting should be optional. You need to test if `.Values.deployment.hostNetwork` exists, then use the setting if it does. Keeps the yaml cleaner.